src/test/resources/features/DepartureFE/AmendDeclaration.feature

```diff
@@ -33,6 +33,8 @@ Feature: End to end journey for amending a departure declaration
     And I should see COMPLETED status for route details on the 'Declaration summary' page
 
     And I click on the Edit transport details link on the 'Declaration summary' page
+    And I choose radio option No on the 'Do you want to add identification for this vehicle?' page
```

Comment:
You shouldn't need to submit the check your answers page as all the data should be in the json file that is passed in at the start of this test. If that includes the answer to this question then you can go straight in and hit the change link.